<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudSubespecie</class>
 <widget class="QMainWindow" name="CrudSubespecie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>793</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administraci√≥n de subespecies</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 134, 137)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbMain">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>1250</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1250</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="gbActivity">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>69</y>
       <width>711</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Control de subespecies</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbAltas">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Altas</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbBusqueda">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbModificar">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="gbAltas">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>271</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Altas</string>
     </property>
     <widget class="QLabel" name="lbSubespecie">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Subespcie</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbIdSubespecie">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Id Subespcie</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneIdSubespcie">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneSubespecie">
      <property name="geometry">
       <rect>
        <x>129</x>
        <y>110</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbAgregar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 190, 151);</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbBuscar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>741</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Buscar</string>
     </property>
     <widget class="QLabel" name="lbSubQuery">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Subespecie</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneQuery">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>551</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvQuery">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>661</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbModificar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>460</y>
       <width>1041</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Modificar</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbIdSubModificar">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Id Subespcie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneIdSubModificar">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbBuscar">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(120, 190, 151);</string>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbSubModificar">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Subespcie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lneSubModificar">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbModificar">
        <property name="font">
         <font>
          <family>Raleway</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(120, 190, 151);</string>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>30</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images.qrc">:/Images/tuIcone.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbOut2">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>80</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 190, 151);</string>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1375</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lbBackground</zorder>
   <zorder>gbMain</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOut2</sender>
   <signal>clicked()</signal>
   <receiver>CrudSubespecie</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1000</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>1131</x>
     <y>731</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
